src/lumigo_tracer/wrappers/aiohttp/aiohttp_wrapper.py

```diff
@@ -0,0 +1,87 @@
+from lumigo_tracer.lumigo_utils import lumigo_safe_execute, get_logger, concat_old_body_to_new
+from lumigo_tracer.libs.wrapt import wrap_function_wrapper
+from lumigo_tracer.spans_container import SpansContainer
+from lumigo_tracer.wrappers.http.http_data_classes import HttpRequest
+from lumigo_tracer.wrappers.http.sync_http_wrappers import add_request_event, update_event_response
+
+try:
+    import aiohttp
+except Exception:
+    aiohttp = None
+
+LUMIGO_SPAN_ID_KEY = "_lumigo_span_id"
+trace_config = None
+
+
+def aiohttp_session_init_wrapper(func, instance, args, kwargs):
+    with lumigo_safe_execute("aiohttp aiohttp_session_init_wrapper"):
+        traces = kwargs.get("trace_configs") or []
+        traces.append(trace_config)
```

Comment:
Are we sure that the type of `traces` is always a `list`?

Reply:
That's the api, but I'll add protection 🤔